source/DailyOverview/DailyOverview.html

```diff
@@ -10,8 +10,13 @@
     <body>
         <div id="header">
             <h1 id="monthView">
-                <a href="../MonthlyOverview/MonthlyOverview.html" id="words">
-                </a>

```

Comment:
You may need to pull from main as I think someone maybe Hunter, changed the code so there isn't CSS with the HTML which you branch still has